lib/domain/LogStorage.js

```diff
@@ -61,6 +62,7 @@ class GelfLogStorage {
     for (let i = 0; i < this.gelfClients.length; i += 1) {
```

Comment:
could you change ` for (let i = 0; i < this.gelfClients.length; i += 1) {` to ` for (let gelfClient of this.gelfClients) {`

Reply:
Sure.